CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Contributor Code of Conduct
+
+This project adheres to No Code of Conduct.  We are all adults.  We accept anyone's contributions.  Nothing else matters.
```

Comment:
A useless file. Remove it.

Reply:
@Sammers21 GitHub doesn't think so, it asks to add such file
![scrot-2020-07-13-12:08:54](https://user-images.githubusercontent.com/1524988/87286615-ba551a00-c501-11ea-8d9a-fc6e26240aed.png)

Reply:
@g4s8, A code of conduct is a set of rules outlining the norms, rules, and responsibilities or proper practices of an individual party or an organisation. We work with zerocrat and have specific rules. Mention them in the file. Now the file does not bring any value.

Reply:
@Sammers21 we don't work by Zerocracy rules, only users from 0crat.com do. But they are aware about these rules without this file.